tasks/main.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: Install required pakages
+  package:
+    name: "{{ item }}"
+    state: present
+  with_list: "{{ capra_lvm_filesystem_packages }}"
```

Comment:
YUM on CentOS/RHEL can be rather slow. Please test whether the impact of invoking the module twice instead of once (by passing the list directly) isn't significant.